javascript/react/fundamentals/what-are-props.md

```diff
@@ -28,45 +28,56 @@ aspects:
 
 ---
 
-# What are `props`?
+# What are props?
 
 ---
 ## Content
 
-As discussed before, `component`s can take input in a similar way to a function, via `props`.
+As discussed before, components can take input in a similar way to a function, via `props`.
 
-`props` is a plain `JS` object accessible via `this.props`, its properties representing the arguments of a `component`.
+`props` is a plain JS object accessible via `this.props`, its properties representing the arguments of a component.
 
-An important aspect of the `props` object is that it is **immutable**. That means `props` can't and shouldn't be changed within a `component`, but only from its *parent*(s).
+An important aspect of the `props` object is that it is **immutable**. That means `props` can't and shouldn't be changed within a component, but only from its *parent*(s).
 
 Consider the component:
 ```jsx
 class Greeting extends React.Component {
   render() {
-    return <h1>Enki greets
-      {this.props.name}</h1>;
+    return (
+      <h1>
+        Enki greets
+        {this.props.name}
+      </h1>
+    );
   }
 }
 ```
 
-If we want to `render` it, the `element` must specify the value of `name` prop:
+Note that if the value of your `return` extends over multiple lines it must be enclosed in parenthesis.
+
+If we want to `render` it, the element must specify the value of `name` prop:
+
 ```jsx
-const element = <Greeting
+const el = <Greeting
```

Comment:
Write this snippet as:

```jsx
const el = (
  <Greeting name="John" />;
);
```

Reply:
For this example, we didn't pass a `name` prop on purpose because we are showing the user how default `props` behave. Should I write it as:

```js
const el = (
  <Greeting  />;
);
```

Reply:
Disregard, I was reading the wrong line.